### PR TITLE
feature(TL-309): Repair list component

### PR DESCRIPTION
**Pull request by kymmeh**

### CODE REVIEW

File: lib/test-utils.ts [hunk 1]
```diff
@@ -32,9 +34,44 @@ export const mockCommentsV2 = Array.from({ length: 20 }).map((_, index) =>
...
+    tradeCode: "GS",
+    tradeDescription: "Domestic gas: servicing",
+    status: "In Progress",
+  },
+];
+
+export const getRepairsV1 = (data: any = mockRepairs, code = 200) =>
+  rest.get(`${config.repairsHubApiUrl}/worksOrders`, (req, res, ctx) => {
+    return res(ctx.status(code), ctx.json(data));
+  });
```
Comment (kymmeh) [inline lib/test-utils.ts:69]: I'm in the process of putting a PR in to move the mock/handler into test-utils.

File: lib/components/card-list/card-list-rows.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+export interface CardListRowsProp {
+  rows: CardListRow[];
+}
+
+export const CardListRows = ({ rows }: CardListRowsProp) => {
+  return (
+    <dl className="mtfh-card-list-rows">
+      {rows.map((row: CardListRow, index) => (
+        <div key={index} className="mtfh-card-list-rows__row">
```
Comment (cloudratha) [inline lib/components/card-list/card-list-rows.tsx:17]: There is a SummaryList component that is very similar to this.

File: lib/components/card-list/card-list-item.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import React, { ComponentPropsWithoutRef } from "react";
+import cn from "classnames";
+
+import "./styles.scss";
+
+export const CardListItem = ({
```
Comment (cloudratha) [inline lib/components/card-list/card-list-item.tsx:6]: I would rename this to just Card, so its not so coupled to the CardList component

Comment (cloudratha) [reply lib/components/card-list/card-list-item.tsx:6]: Also please look at the isomorphic approach for this component.

**Review by cloudratha — APPROVED** [review]

File: lib/components/card/card.test.tsx [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+test("Card renders", () => {
+  const rows = [
+    { value: "value1", label: "Label 1" },
+    { value: "value2", label: "Label 2" },
+  ];
+  const { container } = render(
+    <Card>
+      <CardRows rows={rows} />
+      <CardBreak />
+      <p>Additional content</p>
```
[the remark this answers is not in the document]
Comment (kymmeh) [reply lib/components/card/card.test.tsx:17]: This is a test

**Review by RVerma90 — APPROVED** [review]

**Pull request merged**